### ISSUE TITLE
BUG: Unable to select mariadb-dump for maria-db backups

### ISSUE THREAD

**Issue by origamifreak2** [beekeeper-studio/beekeeper-studio#2034]
**Is your feature request related to a problem? Please describe.**
I am unable to create a backup of the database using the "Create a database backup" tool because I only have mariadb-dump instead of mysqldump.

**Describe the solution you'd like**
The "Create a database backup" tool should allow the use of mariadb-dump.

**Describe alternatives you've considered**
Installing mysql alongside mariadb.

**Additional context**
I have the mariadb-dump executable installed via homebrew at `/usr/local/Cellar/mariadb/11.1.2/bin/mariadb-dump` and it is symlinked at `/usr/local/bin/mysqldump`. When I select the symlink, beekeeper studio shows the mariadb-dump location. I'm assuming this is why it won't allow me to do a DB export.

https://github.com/beekeeper-studio/beekeeper-studio/assets/2578041/72726d9d-3371-44c2-96ab-9a18cffea19b

**Database type, if appropriate**
MariaDB

**Comment by rathboma** [comment]
Great point, sorry about that! We'll make sure you can select mariadb-dump :-)
